Source/ACE/Entity/Landblock.cs

```diff
@@ -466,6 +481,28 @@ private void HandleGameAction(QueuedGameAction action, Player player)
         {
             switch (action.ActionType)
             {
+                case GameActionType.TalkDirect:

```

Comment:
So, about TalkDirect....  as we discussed, this is a hack.  I don't see where you're enqueueing this ActionType at all.  Is it even possible to hit this code block?